src/js/components/CheckBoxGroup/doc.js

```diff
@@ -0,0 +1,53 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils';
+
+export const doc = CheckBoxGroup => {
+  const DocumentedCheckBoxGroup = describe(CheckBoxGroup)
+    .availableAt(getAvailableAtBadge('CheckBoxGroup'))
+    .description('A group of CheckBoxes.')
+    .usage(
+      `import { CheckBoxGroup } from 'grommet';
+        <CheckBoxGroup />`,
+    )
+    .intrinsicElement('div');
+
+  DocumentedCheckBoxGroup.propTypes = {
+    checked: PropTypes.arrayOf(PropTypes.string).description(
+      `An array of the checked options. If options is provided as an object, the
+      checked array will be the strings of the checked option valueKey.`,
+    ),
+    disabled: PropTypes.bool
+      .description(`Disables all options.`)
+      .defaultValue(undefined),
+    labelKey: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.func,
+    ]).description(
+      `When the options array contains objects, this property indicates how
+        to determine the label of each option. If a string is
+        provided, it is used as the key to retrieve each option's label.`,
+    ),
+    name: PropTypes.string.description(
```

Comment:
How does this work when each CheckBox should have a `name`? I don't think `<div />` elements can have `name`, just input-like things. I think what I mean to suggest is that `name` is used for the Form context but not as a DOM attribute.

Reply:
I agree, `name` isn't an input element.

Reply:
Overall I agree. 
Though, the `name` isn't being assigned on the `div` level but on the individual `input` (Checkbox) level. why would that be a problem?

Reply:
I think it will be a problem. A good way to think about it is how would `<form action="" />` behave if the default browser behavior submitted it, via a POST. The form data would pick out the contained values keyed by "name" attribute of the input elements.

Reply:
Good observation. 
I'll clean up the documentation and code with the direction of this comment (I'll remove the actual assignment of `name` from CheckBox and only use `name` when calling the Form context).
The user will still be available to assign the `name` prop per each CheckBox via the `options` object.